README.md

```diff
@@ -71,3 +71,23 @@ The [homepage](https://github.com/ishepard/pydriller) and [documentation](https:
 for `PyDriller`.
 
 The available attributes can be found at their homepage.
+
+## Steps to run `count_code_lines.py`
+
+- Must be in the `src` folder
+- Run the python file using `pipenv run python print_table.py`
+- Paste the URL or local path of the repository
+- If the current repository is meant to be selected, enter `../` as the path.
+    This will cause the program to take the directory right outside of the
+    `src` folder
+- __NOTE__: the program only displays statistics for the default master branch
+    of the repository if run using repository URL. However, it would display
+    the checked out branch statistics if the local path was provided. This is
+    a current limitation of `PyDriller`
```

Comment:
I think this should just go into the docstring for the module instead of here, but that's just my personal opinion.

Reply:
Following these instructions currently gives a traceback error.

Reply:
I agree, I think the docstring my be a more suitable place to show this information

Reply:
Yes the `docstring` would be a better place to show this information, which is why I moved it there.

Reply:
I just put additional instructions in the `README.md` file. Let me know how they work!